src/app/games/awale/AwaleRules.ts

```diff
@@ -122,31 +122,31 @@ export class AwaleRules extends Rules<AwaleMove, AwalePartSlice, AwaleLegalitySt
         // does not make the capture nor verify the legality of the move
         // return the coord of the last case the move got down
 
-        // iy et ix sont les cases initiales
+        // iy and ix are the initial squares
         const ix: number = x;
         const iy: number = y;
-        // à retenir pour appliquer la règle de la jachère en cas de tour complet
+        // to remeber in order not to sow in the starting case if we make a full turn
         let inHand: number = board[y][x];
-        board[y][x] = 0; // on vide la case
+        board[y][x] = 0; // empty the case
```

Comment:
squares, house, case, we kind of don't care about comments but if you feel like leaving them uniform while translating them, bonus point :D !

Also, reme**m**ber

Reply:
c'est vrai que ce genre de commentaire pourrait partir, c'pas ouf `doBlibli(); // we do blibli` :D

Reply:
Oui c'est pas le plus utile.

Reply:
j'ai uniformisé pour space (ce qui me semble avoir plus de sens à l'awalé, même si house serait ptet mieux?), et enlevé ce commentaire ci